src/components/Header/HomePageHeader.tsx

```diff
@@ -30,7 +30,7 @@ const HomePageHeader = (props: { indicatorsLinkOnClick: () => void }) => {
               Our map shows risk levels for <strong>50 states</strong> and{' '}
               <strong>3,200 counties</strong> using{' '}
               <ClickableCopy onClick={props.indicatorsLinkOnClick || noop}>
-                5 key metrics
+                3 key metrics
```

Comment:
Do we want to have a note about this changing? I'm not sure if we explicitly have a blog post or similar explainer about why it went from 5 to 3

Reply:
I have mentioned this to Debbie.